Classes/Command/CheckContentEscapingCommand.php

```diff
@@ -252,7 +252,8 @@ protected function discoverTemplateFiles(): array
     {
         // All extensions in local extension directory
         $activeExtensions = array_filter($this->packageManager->getActivePackages(), function ($package) {
-            return strpos($package->getPackagePath(), Environment::getExtensionsPath()) === 0;
```

Comment:
There is actually API for that in the package, which delivers more accurate results. `!$package->isFrameworkType()`, which I think is exactly what you want. It's only available since TYPO3 11 though.